### DISCUSSION TITLE
Will we have an updated look for the Vue Test Utils website as happend with Pinia and Vue Router?

### DISCUSSION THREAD

**Discussion by nazarepiedady** [vuejs/test-utils#1988] [General]
All who use Vue.js at an early stage know how old the Vue Test Utils website look,
at least compared to its siblings, So I would like to discuss the strategy to update
the website of Vue Test Utils to pair with its ecosystem siblings.

@lmiller1990 @cexbrayat @afontcu @pikax

**Comment by cexbrayat** [comment]
I think we would love it, but AFAIK, none of us is working on it.
Would you be interested in contributing to that?

**Reply by nazarepiedady** [reply]
@cexbrayat I will do my best to help and to do relevant contributions.

**Reply by lmiller1990** [reply]
Sure thing, find out what those projects are doing and do it here - can we just grab their Vitepress theme?

**Reply by nazarepiedady** [reply]
@lmiller1990 I think that would be a good idea.
